### DISCUSSION TITLE
Best unblocked sites

### DISCUSSION THREAD

**Discussion by Inglan2** [3kh0/ext-remover#1041] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm making a website called schooldocs with unblocked sites, exploits, and more stuff for school. I need some unblocked games sites to put on it.

If you know of any good sites put them below and I might add.

Tell me if they have ads, are on Google sites, have any AI features, and if you recommended them. it makes it easier when I'm adding them.

**Comment by Blobby-Boi** [comment, marked as answer]
theres monkeygg which is like a cool site its not on google sites has a lot of games but no ai features and I definitely recommend it
also learn sci is cool
also if you're gonna add exploits make sure to get the html file for caub and sipe and stuff and put it there cuz its blocked for many people
also i made a game site last year but it kinda sucks but [here it is](https://sites.google.com/view/edu-blobe/)
it has chatgpt in it and its on google sites with no ads but i dont recommend it that much cuz its a bit laggy

**Reply by AshtonDavies** [reply]
> also if you're gonna add exploits make sure to get the html file for caub and sipe and stuff and put it there cuz its blocked for many people

[CAUB](https://minies.pages.dev/caub)
[SIPE](https://minies.pages.dev/sipe)
[NPPE](https://minies.pages.dev/nppe)

I found these on the web at one point.

**Reply by Inglan2** [reply]
Nice. pages.dev is unblocked at my school so i can download

**Reply by Inglan2** [reply]
I will save them and upload. I think mkdocs material supports HTML.
